manager/integration/tests/test_basic.py

```diff
@@ -2785,10 +2786,6 @@ def test_backup_lock_deletion_during_restoration(client, core_api, volume_name,
 
     _, b = common.find_backup(client, std_volume_name, snap1.name)
     client.create_volume(name=restore_volume_name, fromBackup=b.url)
-    wait_for_volume_condition_restore(client, restore_volume_name,
```

Comment:
We don't need to wait for the RestoreInProgress condition?

Reply:
We have ```wait_for_volume_restoration_start``` check at the next line, so this was not required.